release-notes/opensearch-anomaly-detection-dashboards.release-notes-3.0.0.0.md

```diff
@@ -0,0 +1,19 @@
+## Version 3.0.0.0 Release Notes
+
+Compatible with OpenSearch Dashboards 3.0.0.0
+
+### Features
+- Implmentation of contextual launch ([#1005](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/1005))
```

Comment:
this has already went out with previous release, do we still need to include it in the 3.0 release notes?